package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "bellhop-iframe",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
Let's mark this as a minor update, rather than patch (since it's a new feature and not a bugfix).